keel-ec2-plugin/src/test/kotlin/com/netflix/spinnaker/keel/api/ec2/DeserializationInjectionTests.kt

```diff
@@ -16,13 +18,20 @@ import strikt.assertions.isNotNull
 import strikt.assertions.isSuccess
 
 @SpringBootTest(
-  classes = [Ec2JsonTestConfiguration::class],
+  properties = [
+    "keel.plugins.ec2.enabled = true",
+    "spring.liquibase.enabled = false" // TODO: ignored by kork's SpringLiquibaseProxy
+  ],
+  classes = [KeelApplication::class],
   webEnvironment = NONE
 )
 internal class DeserializationInjectionTests {
 
   @Autowired
-  lateinit var mapper: ObjectMapper
+  lateinit var mapper: YAMLMapper
+
+  @Autowired
+  lateinit var liquibase: SpringLiquibase
```

Comment:
Why are you wiring this in? Doesn't look like it's used

Reply:
Left-over from trying to disable migrations. Will remove!